packages/components/src/HtmlRenderer/Models.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+// html-to-react uses htmlparser to parse input and each 
+// node in the output tree follow htmlparser's specific format:
+// https://www.npmjs.com/package/htmlparser
+
+exports.getRawImpl = (node) => {
+    if (node && node.raw)
+      return node.raw;
+
+    return null;
+}
+
+exports.getDataImpl = (node) => {
+    if (node && node.data)
+      return node.data;
+
+    return null;
+}
+
+exports.getTypeImpl = (node) => {
+    if (node && node.type)
+      return node.type;
+
+    return null;
+}
+
+exports.getNameImpl = (node) => {
+    if (node && node.name)
+      return node.name;
+
+    return null;
+}
+
+exports.getAttribsImpl = (node) => { 
```

Comment:
Remove trailing white space from this line and from the rest of exports in this file too.